.github/on_issue_opened.yml

```
@@ -0,0 +1,32 @@
+name: On issue opened
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  respond-to-issue:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Post Automated Response
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const issueNumber = context.payload.issue.number;
+
+            // Automated response message
+            const message = `
+              Please note that GitHub issues are meant for developer support!
```

I'd consider removing the exclamation mark, so it doesn't sound that "aggresive"

```suggestion
              Please note that GitHub issues are meant for developer support
```